fixtures/getCommits.json

```diff
@@ -339,7 +339,7 @@
           "email": "noreply@github.com",
           "date": "2021-08-24T21:19:53Z"
         },
-        "message": "docs: Update inter-linking to match Canvas Site (#1220)\n\n\r\nCo-authored-by: James Fan <james@somewhatstrange.com>\r\nCo-authored-by: Alex Nicholls <anicholls3@gmail.com>",
+        "message": "docs: Update inter-linking to match Canvas Site (#1220)\n\n\r\nCo-authored-by: @jamesfan <james@somewhatstrange.com>\r\nCo-authored-by: @anicholls <anicholls3@gmail.com>",
```

Comment:
I manually updated these to match the format of commit messages. The `@` indicates the user has a login, otherwise the Release Notes will list the name as a string and not an account.